Map updates and fixes, plus hardsuit icons

mwerezak opened the pull request:
* Adds a suit cycler to the mercenary base gear storage area.
* Also adds a pair of STS-35s + ammo. Does slightly more damage 30 instead of 25 in exchange not being storeable in a backpack. I figured if there's now a chance of high-powered battle rifles spawning in the sec armoury, it's more than fair (although I really thing the armoury spawner lists need to be revised).
* Reuses the mercenary voidsuit icons for the crimson hardsuit species sprite sheets.
* Adds a gas meter to the mercenary shuttle airlock air reserve, so they know when it's run out.
* Fixes several space vent pumps that had bad pressure_check values, which prevented them from working.
* Adds a t-valve connection between the special xenobio/flora scrubber pipenet and the main scrubber line.
* Adds an airlock air sensor to the chapel mass driver chamber, so players know when it's safe to open the door again.
* Adds both a heater and freezer to to xenobotany storage to allow xenobotanists to create plant environments at specific temperatures.
* Adds a precision measurement fuel injection system to toxins lab.

![screenshot 2015-02-20 00 27 00](https://cloud.githubusercontent.com/assets/242428/6282218/210f8296-b89f-11e4-8671-2da22c51eb62.png)
![screenshot 2015-02-20 20 14 53](https://cloud.githubusercontent.com/assets/242428/6312176/a3297f08-b93e-11e4-8720-c9250624bdfd.png)
![screenshot 2015-02-20 20 15 09](https://cloud.githubusercontent.com/assets/242428/6312177/a4398032-b93e-11e4-80ab-28ca537a4956.png)

![screenshot 2015-02-20 01 07 47](https://cloud.githubusercontent.com/assets/242428/6282227/44df6358-b89f-11e4-947c-52c62c03d808.png)

PsiOmegaDelta commented:
Having selected Pprple for the isolated ventilation pipes sure seems like a bad choice of mine now.

mwerezak commented:
Just noticed that in the second screenshot the isolation tray pump connects to distro. What.

GinjaNinja32 commented:
>Just noticed that in the second screenshot the isolation tray pump connects to distro. What.

IMO, this should be replaced with two pumps; one from distro to the trays, and one from the trays to scrubbers.
That way xenoflora can fill the trays with air, plant their plants and do their thing, if the gas turns out to be bad they can kill the plant, pump the gas to scrubbers for Engineering to deal with, fill the trays again and start over.

mwerezak commented:
I'm not sure if trays need to be filled with air in the first place...

GinjaNinja32 commented:
I don't know exactly how it works, but they definitely need some way to pull bad gases out, *and* add plain air in without contaminating an entire canister of it.

Zuhayr commented:
They use a linked pipe if the lid is closed and one is available, otherwise they use ambient. They don't fill up.

GinjaNinja32 commented:
Ah, that makes sense. Still need a way to put air into the pipe and pull bad gas out.

mwerezak commented:
Screenshots have been updated.

comma commented:
:+1:

Zuhayr commented:
Cannot automerge.

mwerezak commented:
Merge conflict resolved.

comma commented:
Aaaand again.

mwerezak commented:
Merge conflict resolve, I also added a couple of things, so this might need to be re-approved.

Added the ability for uplink purchases to produce multiple items. Ended up not using it, but I left it in anyways.

Nerfed the mercenary sniper rifle by lowering damage from 80 to 60, and made it uplink purchasable.
